### ISSUE TITLE
`...attr` which replaces now deprecated `${attr}` breaks class Array syntax

### ISSUE THREAD

**Issue by gunjam** [marko-js/marko#1007]
### Details

The current version of Marko gives me the following warning in the console: `${attrs} is deprecated, use ...attrs instead`

However when I refactored my component to use `...attrs` it broke the `class=['class1', 'class2']` syntax for the element I was spreading attributes into.

#### Example

Simple input component:
```marko
$ const extraAttrs = input['*'] || {};

<input class=['form-control', input.error && 'form-control-error']
  name=input.name type="text" ...extraAttrs/>
```

In template:
```marko
<simple-input name="full-name" error=false maxlength=5/>
```

### Expected Behavior

Should render:
```html
<input class="form-control" name="full-name" type="text" maxlength="5"/>
```

### Actual Behavior

Renders:
```html
<input class="[&quot;form-control&quot;,null]" name="full-name" type="text" maxlength="5"/>
```

class Array is JSON stringified.

<details><summary>Additional Info</summary>

### Your Environment

* Firefox developer edition 59
* macOS High Sierra 10.13.3
* node v9.5.0
</details>

**Comment by gunjam** [comment]
Ta!

**Comment by johannesmutter** [comment]
Same issue with lasso
`WARNING!!
${attrs} is deprecated, use ...attrs instead
  at node_modules/lasso/head-118-3-26-33519-xjraht.l20k.marko:1:0`

**Comment by streamgao** [comment]
same issue here.  any updates?

**Comment by planetarydev** [comment]
@austinkelleher Hey Austin, any news on this issue? The same warning we got in markojs 4.9.0 and 4.10.0.

**Comment by mlrawlings** [comment]
Fixed in `4.11`
